demos/text_detection_demo/src/cnn.cpp

```diff
@@ -65,18 +65,17 @@ void Cnn::Init(const std::string &model_path, Core & ie, const std::string & dev
     infer_request_ = executable_network.CreateInferRequest();
     // ---------------------------------------------------------------------------------------------------
 
-    // --------------------------- Preparing input -------------------------------------------------------
-
-    /* Resize manually and copy data from the image to the input blob */
-    Blob::Ptr input = infer_request_.GetBlob(input_name);
-    input_data_ = input->buffer().as<PrecisionTrait<Precision::FP32>::value_type *>();
-
     is_initialized_ = true;
 }
 
 InferenceEngine::BlobMap Cnn::Infer(const cv::Mat &frame) {
     std::chrono::steady_clock::time_point begin = std::chrono::steady_clock::now();
 
+    /* Resize manually and copy data from the image to the input blob */
+    InferenceEngine::LockedMemory<void> inputMapped =
+        InferenceEngine::as<InferenceEngine::MemoryBlob>(infer_request_.GetBlob(input_name_))->wmap();
+    input_data_ = inputMapped.as<float *>();
```

Comment:
Seems like `input_data_` could now be turned into a local variable.

Reply:
The trailing underscore at the end of the name should be removed, since it's not a member variable anymore.